tests/table/test_upsert.py

```diff
@@ -25,6 +25,9 @@
 from pyiceberg.exceptions import NoSuchTableError
 from pyiceberg.schema import Schema
 from pyiceberg.table import UpsertResult
+from pyiceberg.table.upsert_util import create_match_filter
+from pyiceberg.expressions import And, Reference, EqualTo
+from pyiceberg.expressions.literals import LongLiteral
```

Comment:
The linter is very picky :)
```suggestion
from pyiceberg.expressions import And, EqualTo, Reference
from pyiceberg.expressions.literals import LongLiteral
from pyiceberg.schema import Schema
from pyiceberg.table import UpsertResult
from pyiceberg.table.upsert_util import create_match_filter
```